modules/otel-config-instrumentation.adoc

```diff
@@ -98,6 +98,105 @@ spec:
 
 |===
 
+[id="otel-instrumentation-options_exporter_{context}"]
+== Exporter configuration
+
+Although the `Instrumentation` custom resource supports setting up one or more exporters per signal, auto-instrumentation configures only the OTLP Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
```

Comment:
```suggestion
Although the `Instrumentation` custom resource supports (CR) setting up one or more exporters per signal, auto-instrumentation configures only the OpenTelemetry Protocol (OTLP) Exporter. So you must configure the endpoint to point to the OTLP Receiver on the Collector.
```
I would define the acronym here. You could skip the definition, then, in the No 1 callout (or leave it).

Reply:
Thank you for the suggestion.
I'm dropping the definition because the OTLP is a well-known abbreviation for the target audience (think of JSON or HTTP).